tools/who_what_benchmark/tests/constants.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+import os
+import tempfile
+
+
+WWB_CACHE_PATH = Path(os.path.join(os.environ.get('OV_CACHE', tempfile.TemporaryDirectory().name), 'wwb_cache'))
```

Comment:
I think it's better reuse genai approach: https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/utils/constants.py#L39